tests/test_dataclasses_config.py

```diff
@@ -0,0 +1,67 @@
+import pytest
```

Comment:
probably easier to keep all this in `test_dataclasses.py`

Reply:
Ok